docs/static/breaking-changes.asciidoc

```diff
@@ -70,3 +70,32 @@ Starting with the 2.0 release of Logstash, the default value of the `filter_work
 plugins is half of the available CPU cores, instead of 1. This change increases parallelism in filter execution for
 resource-intensive filtering operations. You can continue to use the `-w` flag to manually set the value for this option,
 as in previous releases.
+
+
+[float]
+=== Ruby filter Changes
+
+With the migration to the Java Event we have changed how we can access the internal data, the object doesn't return a
```

Comment:
I suggest changing this section to say: "With the migration to the Java Event (https://github.com/elastic/logstash/issues/4191[Issue 4191]), we have changed how you can access internal data. The Event object no longer returns a reference to the data. Instead, it returns a copy. This might change how you do manipulation of your data, especially when working with nested hashes."